src/coreComponents/schema/docs/AcousticSEM_other.rst

```diff
@@ -1,20 +1,20 @@
-sourceIsLocal             integer_array                                                                                                                                       Flag that indicates whether the source is local to this MPI rank        
-sourceNodeIds             integer_array2d                                                                                                                                     Indices of the nodes (in the right order) for each source point         
-sourceValue               real64_array2d                                                                                                                                      Source Value of the sources                                             
-LinearSolverParameters    node                                                                                                                                                :ref:`DATASTRUCTURE_LinearSolverParameters`                             
-NonlinearSolverParameters node                                                                                                                                                :ref:`DATASTRUCTURE_NonlinearSolverParameters`                          
-========================= =================================================================================================================================================== ======================================================================= 
+========================= =========================================================================================================================================== ======================================================================= 
+Name                      Type                                                                                                                                        Description                                                             
+========================= =========================================================================================================================================== ======================================================================= 
+indexSeismoTrace          integer                                                                                                                                     Count for output pressure at receivers                                  
+maxStableDt               real64                                                                                                                                      Value of the Maximum Stable Timestep for this solver.                   
+meshTargets               geosx_mapBase< string, LvArray_Array< string, 1, camp_int_seq< long, 0l >, int, LvArray_ChaiBuffer >, std_integral_constant< bool, true > > MeshBody/Region combinations that the solver will be applied to.        
```

Comment:
The string type name changes tend to be buried in these maps.  I've added a simple regex to convert the compiler-dependent values to be string (tested on clang and gcc)